client/src/components/Navbar/Navbar.tsx

```diff
@@ -60,18 +60,20 @@ export default function Navbar({ user, profile, logout }: Props): JSX.Element {
 
   const UserNav = () => (
     <Grid>
-      <Link to="/checkout" className={classes.link}>
-        <Button color="primary" className={`${classes.btn} ${classes.signupbtn}`} variant="contained">
-          Checkout
-        </Button>
-      </Link>
+      {profile?.isDogSitter ? (
+        <Link to="/requests" className={classes.link}>
```

Comment:
You can merge Link and Button. 
Eg: "<Button component={Link} to='/requests'>Request</Button>"